commands/gist.go

```diff
@@ -46,19 +52,21 @@ gist show <ID> [<FILENAME>]
 
 hub(1), hub-api(1)
 `,
-		KnownFlags: "\n",
 	}
 
 	cmdShowGist = &Command{
-		Key:        "show",
-		Run:        showGist,
-		KnownFlags: "\n",
```

Comment:
can you explain the newline passed to known flags as opposed to not listing it? I got that from `pr.go`...